docs/pages/api-reference/liveblocks-node.mdx

```diff
@@ -1585,6 +1570,57 @@ await liveblocks.deleteRoomNotificationSettings({
 });
 ```
 
+### Notifications
+
+#### Liveblocks.getInboxNotification [#get-users-userId-inboxNotifications-inboxNotificationId] [@badge=Beta]
+
+Returns a user’s inbox notification. This is a wrapper around the
+[Get Inbox Notification API](/docs/api-reference/rest-api-endpoints#get-users-userId-inboxNotifications-inboxNotificationId).
+
+```ts
+const inboxNotification = await liveblocks.getInboxNotification({
+  userId: "steven@example.com",
+  inboxNotificationId: "in_3dH7sF3...",
+});
+
+// { id: "in_3dH7sF3...", kind: "thread", ... }
+// or { id: "in_3dH7sF3...", kind: "$yourKind", ... }
+console.log(inboxNotification);
+```
+
+#### Liveblocks.triggerInboxNotification [#trigger-inbox-notification] [@badge=Beta]
+
+Triggers an inbox notification based on an activity that happened in your
+system. This is a wrapper around the
+[Trigger Inbox Notification API](/docs/api-reference/rest-api-endpoints#post-inbox-notifications-trigger).
+
+`kind` must start with a `$`.
+
+`activityData` can have `string`, `number`, or `boolean`.
```

Comment:
I would also put this with "`kind` must start with a `$`." above, not as a new paragraph.

```suggestion
Properties of `activityData` can have `string`, `number`, or `boolean` values.
```